src/panel.tsx

```diff
@@ -14,7 +14,9 @@ const Iframe = styled.iframe({
 });
 
 function parseShareURL(url: string): string | void {
-  return url.split(/share.(?:go)?abstract.com\//)[1];
+  const parsedUrl = new URL(url);
+  const pathSegments = parsedUrl.pathname.split('/');
+  return pathSegments[pathSegments.length - 1];
```

Comment:
It’s important that this continues to only match a those specific hostnames

Since this function only needs to parse the shareId, maybe change `split` to `match` to ignore trailing query string?

Reply:
Ah thanks, I missed that. Have resolved now.